classes/image/imagemagick.php

```diff
@@ -158,6 +158,14 @@ protected function _mask($maskimage)
 		$this->exec('convert', $command);
 	}
 
+	protected function _blur($radius, $sigma)
+	{
+		extract(parent::_blur($radius, $sigma));
+
+		$image = '"'.$this->image_temp.'"';
+		$this->exec('convert', $image.' -blur '.$radius.'x'.$sigma.' '.$image);
```

Comment:
`$radius.'x'.$sigma` should be quoted for shell use, or casted to actual floats somewhere (also be careful of the tostring behavior on floats)